### DISCUSSION TITLE
Getting Vue dev tools running

### DISCUSSION THREAD

**Discussion by christianlupus** [nextcloud/cookbook#1317] [General]
Hello.

Having seen the comments [about Vue dev tools](https://github.com/nextcloud/cookbook/issues/1117#issuecomment-1197727364), I wanted to ask who is successfully able to access the dev tools.

This might be interesting for you, @MarcelRobitaille.

**Comment by christianlupus** [comment]
Here I found a hint related to firefox: https://github.com/nextcloud/nextcloud-vue#using-vue-devtools-in-firefox

**Reply by MarcelRobitaille** [reply]
I'll give it a shot, thanks

**Reply by christianlupus** [reply]
Also make sure, that hmr_enabler is active.
